common/djangoapps/custom_settings/__init__.py

```diff
@@ -1 +1,5 @@
+"""
+Custom settings is an app, and part of course settings in studio, which control specific course functionality. Course
```

Comment:
Please consider rephrasing this to `The custom settings app is a part of course settings in studio, which control**s** ...`